client/my-sites/stats/site.jsx

```diff
@@ -194,6 +197,7 @@ export default connect(
 	state => {
 		const siteId = getSelectedSiteId( state );
```

Comment:
does this component actually need a siteId? Or is it just the `stats-navigation/index.jsx` component in which case I'd say you should move this selector to the connect within that component.